ui/lib/core/addon/utils/advanced-secret.js

```diff
@@ -30,7 +30,12 @@ export function obfuscateData(obj) {
   for (const key of Object.keys(obj)) {
     if (Array.isArray(obj[key])) {
       newObj[key] = obj[key].map(() => '********');
-    } else if (typeof obj[key] === 'object') {
+    } else if (typeof obj[key] === 'object' && obj[key] === null) {
```

Comment:
You can just check if it === null 
```suggestion
    } else if (obj[key] === null) {
```
and then below just check if it's type object

Reply:
Thank you!